dll/win32/msgina/gui.c

```diff
@@ -498,9 +606,11 @@ WelcomeDialogProc(
                 return FALSE;
 
             DlgData_LoadBitmaps(pDlgData);
+            DlgData_MoveControls(hwndDlg, pDlgData);
             return TRUE;
         }
 
+#if 0
```

Comment:
Why #if 0 and not deleting the code enclosed by that? Same question for the other occurences of the same thing.